packages/devextreme-cli/src/applications/application.angular.js

```diff
@@ -30,7 +30,7 @@ async function runSchematicCommand(schematicCommand, options, evaluatingOptions)
 }
 
 async function runNgCommand(commandArguments, evaluatingOptions) {
-    const ngCommandArguments = await hasSutableNgCli() ? [] : ['-p', '@angular/cli@v9-lts'];
+    const ngCommandArguments = await hasSutableNgCli() ? [] : ['-p', '@angular/cli@9.1.12'];
```

Comment:
Do we need to freeze angular cli version in the context of this PR?

Reply:
No, we should revert this change

Reply:
I reverted the change